Tests/DataParser/test_dataParser.py

```diff
@@ -0,0 +1,116 @@
+import os
+import json
+import xmltodict
+import zipfile
+import pandas as pd
+from pathlib import Path
+import pytest
+import sys
+sys.path.append("../../DataParser")
+import filecmp
+import dataParser
+
+testFiles = "./testFiles"
+truth_folder = "./testFiles/truth_data"
+
+__TESTS__ = 4 
+
+def build_testdata():
+	"""
+	This method builds the gold directory structure for the files added
+	in the test folder of the data path. This created directory structure 
+	will be used while comparing the test.
+	"""
+	os.system("rm -rf ./testFiles/truth_data/*")
```

Comment:
Do not call rm -rf commands here. This would not be safe. Try to delete the needed data in a pythonic way or better yet, just create a new folder  to put all the test data for every run. You can maybe create the folder with the current timestamp so that it is unique.